deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -224,8 +224,7 @@ void IsolateShim::PushScope(
   this->contextScopeStack = scope;
 
   // Don't crash even if we fail to set the context
-  JsErrorCode errorCode = JsSetCurrentContext(contextShim->GetContextRef());
-  CHAKRA_ASSERT(errorCode == JsNoError);
+  JsSetCurrentContext(contextShim->GetContextRef());
```

Comment:
That'll work too. I can revert all these CHAKRA_ASSERTs if you feel they are useful.